Lib/venv/scripts/posix/activate.csh

```diff
@@ -16,10 +16,10 @@ setenv PATH "$VIRTUAL_ENV/__VENV_BIN_NAME__:$PATH"
 setenv VIRTUAL_ENV_PROMPT "__VENV_PROMPT__"
 
 
-set _OLD_VIRTUAL_PROMPT="$prompt"
+set _OLD_VIRTUAL_PROMPT="$prompt:q"
```

Comment:
I think you really want to save the old prompt. Otherwise you could have a prompt that grows indefinitely if you activate/deactive the virtual environment (and you would pollute the next prompt unpon deactivating).